src/components/modals/BalancesModal.tsx

```diff
@@ -116,18 +124,20 @@ export default function BalancesModal({
         </Space>
 
         <Modal
-          title="Edit tracked assets"
+          title={t`Edit tracked assets`}
           visible={editModalVisible}
           onCancel={() => setEditModalVisible(false)}
-          cancelText=""
+          cancelText={t`Cancel`}
```

Comment:
ohh interesting. should we be defining this cancelText for every modal so that it can be translated?

we may be able to make use of this for Antd components https://ant.design/docs/react/i18n#Adding-new-language

Reply:
Cool I'll give that a go!

Reply:
we can leave it for another PR for now!